packages/core/src/html/NodeProcessor.js

```diff
@@ -314,7 +317,8 @@ class NodeProcessor {
     return node;
   }
 
-  process(file, content, cwf = file, extraVariables = {}) {
+  process(file, content, cwf = file, extraVariables = {},
+          shouldValidateIntraLinks = this.config.intrasiteLinkValidation.enabled) {
     const context = new Context(cwf, [], extraVariables, {});
```

Comment:
You can leverage the use of the `Context` object to store the `shouldValidateIntralinks`. `Context` has a property called `processingOptions` that is an object intended to store any options for processing, and it looks like it fits for the situation, with the added bonus that you don't need to keep passing on the flag as a parameter.

Reply:
Thanks for the help! I've switched to using `processingOptions` for passing the flag from `process` to `_process`.